packages/editor/src/components/page-attributes/panel.js

```diff
@@ -1,54 +1,182 @@
 /**
  * WordPress dependencies
  */
-import { __ } from '@wordpress/i18n';
-import { PanelBody, PanelRow } from '@wordpress/components';
-
-import { useSelect, useDispatch } from '@wordpress/data';
+import { __, sprintf } from '@wordpress/i18n';
+import { Dropdown, Button } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
 import { store as coreStore } from '@wordpress/core-data';
+import { useMemo, useState } from '@wordpress/element';
+import { decodeEntities } from '@wordpress/html-entities';
+import { __experimentalInspectorPopoverHeader as InspectorPopoverHeader } from '@wordpress/block-editor';
 
 /**
  * Internal dependencies
  */
+import PostPanelRow from '../post-panel-row';
 import { store as editorStore } from '../../store';
 import PageAttributesCheck from './check';
 import PageAttributesOrder from './order';
 import PageAttributesParent from './parent';
 
 const PANEL_NAME = 'page-attributes';
 
+function getTitle( post ) {
+	return post?.title?.rendered
+		? decodeEntities( post.title.rendered )
+		: `#${ post.id } (${ __( 'no title' ) })`;
+}
+
+function PostParentToggle( { isOpen, onClick } ) {
```

Comment:
Should we move these components inside `order.js and parent.js` respectively?

Reply:
Done 👍